Assets/HoloToolkit/Input/Tests/Scripts/FocusedObjectMessageReceiver.cs

```diff
@@ -0,0 +1,31 @@
+﻿using UnityEngine;
+using System.Collections;
+
+/// <summary>
+/// FocusedObjectMessageReceiver class shows how to handle messages sent by FocusedObjectMessageSender.
+/// This particular implementatoin controls object appearance by changing its color when focused.
+/// </summary>
+public class FocusedObjectMessageReceiver : MonoBehaviour
```

Comment:
I totally support both types of changes. Just want to make sure any ambiguity is resolved wither through comments and tests or not having that code.

I suppose the reason this is advantageous is you can send any custom message to gaze at objects and not just OnGazeEnter and OnGazeLeave.